### PR TITLE
Modify all calls to the story/stories endpoint to use the storiesNew endpoint

### PR DESCRIPTION
**Pull request by chukalicious**
# Description

Add a summary that address the following:

- motivation/purpose for the feature

The Gallery will display the respective story titles and descriptions related to the submission. However, the submissions object that will be received from the BE will not contain this information, hence a new api call needs to be made to access the story information. 

- What were those changes

Modify the existing API call to use the new Story endpoint
Imported api call to new stories action file to use it with the action
Created reducer for the new stories to update app state with the action's payload

- What issues were fixed

The app didn't have a method to make a call to this new endpoint and update its state
- Provide the link to the user story it is referencing in Trello board or other supporting document
https://trello.com/c/OZ5K5K1p/463-modify-all-calls-to-the-story-stories-endpoint-to-use-the-new-story-stories

- Loom video: 
[https://www.loom.com/share/5a5401a6730641e8a06ec404f2d4d104](url)

- Please note that at this point the action has not yet been called in the container in which it is going to be used. This is mainly prep work. 

## Commits Checklist

- [x] commits have clear title names
- [x] commit has descriptive message of what has changed
- [x] more small commits than less big commits

### PRs needs to be reviewed by at least 2 team members

- TPL is the 3rd reviewer
- Lots of comments and suggestions, please!

[Pull Request Rubric](https://www.notion.so/1fc04e4fedeb429ba873b7c68d281707?v=74054da7991341c0bf970f39410c43da)

### CODE REVIEW

File: src/api/index.js [hunk 1]
```diff
@@ -215,12 +215,12 @@ const getChildByID = ID => {
...
  * @param {Object} authState necessary for API functionality
- * @param {number} cohortId the cohort id of the respective child
+ * @param {number} storyId the story id of the respective episode
  * @returns {Promise} a promise that resolves to an object containing {DrawingPrompt, ID, Title, URL, and WritingPrompt}
  */
-const getStory = cohortId => {
+const getStory = storyId => {
   try {
-    return apiAuthGet(`/story?cohortId=${cohortId}`, getAuthHeader()).then(
+    return apiAuthGet(`/storyNew/${storyId}`, getAuthHeader()).then(
```
Comment (tonysorensen) [inline src/api/index.js:223]: This all looks good. We should meet with Jake to see if we can just get rid of "storyNew" at this point and make everything "story" but I don't see that as a reason to hold this up.

**Review by tonysorensen — APPROVED** [review]

File: src/state/reducers/storiesNewReducer.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { storiesNew } from '../actions';
```
Comment (venegasgabby76) [inline src/state/reducers/storiesNewReducer.js:1]: I believe that this is going to change with the new schema, however with this we have a backbone of what we will be needing to work with when we get that.

File: src/api/index.js (hunk 1 above)
Comment (venegasgabby76) [reply src/api/index.js:223]: I agree with tony on this one. once we get the new information we will just have to plug that in

**Review by venegasgabby76 — COMMENTED** [review]
Great job importing the correct information into the reducers as well as the api call. this will display the correct information that is need to display the gallery.

File: src/state/reducers/storiesNewReducer.js (hunk 2 above)
Comment (Aaron-TheCreator) [reply src/state/reducers/storiesNewReducer.js:1]: Actually, this resembles the schema change we've already merged for the "storiesNew" table.

**Review by Aaron-TheCreator — APPROVED** [review]
Awesome, incremental work!

File: src/api/index.js (hunk 1 above)
Comment (artofmayhem) [reply src/api/index.js:223]: Nice and clean conversion of from child to episode. easy to understand logic

File: src/state/reducers/storiesNewReducer.js (hunk 2 above)
Comment (artofmayhem) [reply src/state/reducers/storiesNewReducer.js:1]: clean and easy implementation of Redux getStory function dispatching the incoming data from the call

**Review by artofmayhem — COMMENTED** [review]
Overall very clean and well implemented usage of Redux to tie in the new logic for getting the story.  easy to read and well organized

**Pull request merged**